documentation/s3.md

```
@@ -0,0 +1,68 @@
+
+Depending the Object Sorage used, the complete configuration can differ a little bit.
+
+### With django
+
+The S3 python SDK, named [Boto3](https://github.com/boto/boto3/), is used by Docs.
+Since version 1.36.0, they changed the [Data Integrity Protections](https://docs.aws.amazon.com/sdkref/latest/guide/feature-dataintegrity.html) by enabling it by default.
+Some Object Storage implemenation are not compatible with this change and can lead to signature mismatch error or other errors related to the signature.
+
+To fix this issue, two environment variable can be used to have the older default behavior: 
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)